src/devices/sound/mas3507d.h

```diff
@@ -25,7 +25,23 @@ class mas3507d_device : public device_t, public device_sound_interface
 	void reset_sample_count() { total_sample_count = 0; }
 	u32 get_sample_count() const { return total_sample_count; }
 
-	void set_playback_enabled(bool enabled) { playback_enabled = enabled; }
+	void set_playback_enabled(bool enabled) {
```

Comment:
Perhaps this is getting long enough that it should just be moved to the cpp file instead of header.

Reply:
Removed entirely in latest commit, so marking as resolved.